Users/Glob_Patterns.rst

```diff
@@ -123,14 +121,10 @@ brackets. Parentheses that have no match are ignored as well as
     False
```

Comment:
Why isnt `[!]` an error, or a noisy warning?
It is confusing to list it here, without explanation that this is invalid syntax.